### ISSUE TITLE
Latest changes to nginx ingress controller is not pushed as a docker image

### ISSUE THREAD

**Issue by dhawal55** [kubernetes/contrib#1974]
Trying to deploy the latest rc.yaml file on master branch fails the liveness/readiness probe. Looks like the /healthz port was changed 12 days ago but the docker image is couple months old

**Comment by aledbf** [comment]
@dhawal55 the current code is the correct one.
(we rollback a PR that used references to code in the next version)

**Comment by dhawal55** [comment]
@aledbf I'm trying to deploy the current code and readiness/liveness check fails. The ports specified on current rc.yaml does not match the ones the image version `gcr.io/google_containers/nginx-ingress-controller:0.8.3` is listening on.

**Comment by aledbf** [comment]
@dhawal55 please post the output of `kubectl describe pod <ing pod>` and `kubectl logs <ing pod>`

**Comment by dhawal55** [comment]
duh!! Nginx was failing to load the ingresses:
```
failed to execute nginx -s reload: 2016/11/02 22:20:45 [emerg] 24#24: could not build server_names_hash, you should increase server_names_hash_bucket_size: 64
nginx: [emerg] could not build server_names_hash, you should increase server_names_hash_bucket_size: 64
```

Increasing server-names-hash-bucket-size fixed it. Thanks for your help. Closing this issue.

**Comment by aledbf** [comment]
@dhawal55 in the next version this change is not needed https://github.com/kubernetes/contrib/pull/1596
Sorry for the trouble

**Comment by josdotso** [comment]
@aledbf In which version is the bucket-size fix not needed?  `0.8.3` seems to be the latest tag pushed.  I'm experiencing the bucket-size issue with that tag.  Maybe a new tag needs to be pushed?

https://console.cloud.google.com/kubernetes/images/tags/nginx-ingress-controller?location=GLOBAL&project=google-containers

**Comment by aledbf** [comment]
@josdotso we are in the process to release a new version. Please check https://github.com/kubernetes/ingress/pull/155
